multiversx_sdk_cli/tests/test_cli_all.sh

```diff
@@ -3,15 +3,14 @@
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 USE_PROXY=$1
 
+source ./shared.sh
+source ./test_cli_validators.sh && testAll
+
 testAll() {
     pushd $SCRIPT_DIR
     source ./shared.sh
-    source ./test_cli_contracts.sh && testAll
-    source ./test_cli_dns.sh && testAll
 
     if [ -n "$USE_PROXY" ]; then
-        source ./test_cli_validators.sh && testAll
-        source ./test_cli_tx.sh && testAll
         source ./test_cli_config.sh && testAll
```

Comment:
this can be removed together with the `test_cli_config.sh` file.

Reply:
removed.